### PR TITLE
pkp/pkp-lib#3002 make the plugin sidewide

### PR DESCRIPTION
**Pull request by bozana**
s. https://github.com/pkp/pkp-lib/issues/3002

### CODE REVIEW

File: CustomBlockManagerPlugin.inc.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ function manage($args, $request) {
 	 * @return boolean
 	 */
 	function isSitePlugin() {
-		return !Application::getRequest()->getContext();
+		return true;
```
Comment (asmecher) [inline CustomBlockManagerPlugin.inc.php]: @bozana, won't make this plugin unusable by Managers who aren't also Admins?

Comment (asmecher) [reply CustomBlockManagerPlugin.inc.php]: (Pinging @NateWr on this too, as this reverts his change, I think)

Comment (bozana) [reply CustomBlockManagerPlugin.inc.php]: Ah, @asmecher, I think you are right... Hmmm... So maybe then only not to consider this plugin in the upgrade script i.e. the other solution...

**Comment by bozana** [comment]
This is the wrong solution, thus closing...

**Pull request closed without merging**